services/api/tests/src/test_admin_intersection.py

```diff
@@ -0,0 +1,285 @@
+from unittest.mock import patch, MagicMock, call
```

Comment:
This file seems to be predominantly copy/paste from test_admin_rsu.py, to the point we're even importing the admin_rsu and admin_rsu_data.

Reply:
Good catch, went though and updated all of these tests